src/Credentials/CredentialProvider.php

```diff
@@ -289,8 +289,8 @@ public static function env()
     {
         return function () {
             // Use credentials from environment variables, if available
-            $key = getenv(self::ENV_KEY);
-            $secret = getenv(self::ENV_SECRET);
+            $key = getenv(self::ENV_KEY) ?: $_SERVER[self::ENV_KEY];
```

Comment:
The `$_SERVER` references should check to see if it is set.  Also, we should fall back to false if it isn't. I think you could add a null coalescing operator to what's already here and add `false` on the right side of the operand.